Mathlib/Tactic/Linter/Multigoal.lean

```diff
@@ -61,7 +61,7 @@ There is some overlap in scope between `ignoreBranch` and `exclusions`.
 
 Tactic combinators like `repeat` or `try` are a mix of both.
 -/
-abbrev exclusions : Std.HashSet SyntaxNodeKind := .ofList [
+abbrev exclusions : Std.HashSet SyntaxNodeKind := .ofArray #[
```

Comment:
Local profiling seemed to vaguely indicate that this is more efficient: it's hard to tell for sure.

Reply:
It seems a bit implausible there would be a difference here.